lib/MetaCPAN/Model/Release.pm

```diff
@@ -271,6 +273,34 @@ sub _build_files {
     return \@files;
 }
 
+sub set_changes_file {
```

Comment:
Nitpick here.  This should be `get_changes_file`, since it's only returning the file name and not setting it on the document.  The setter is actually `$document->changes_file()`;